packages-web/slider/tests/e2e/objects/Slider.widget.ts

```diff
@@ -0,0 +1,100 @@
+import page from "../../../../../configs/e2e/src/pages/page";
+
+export enum SliderStyleColor {
+    Default = "rgba(171,226,251,1)",
+    Primary = "rgba(5,149,219,1)",
+    Success = "rgba(118,202,2,1)",
+    Info = "rgba(72,176,247,1)",
+    Warning = "rgba(249,155,29,1)",
+    Danger = "rgba(237,28,36,1)",
+    Inverse = "rgba(37,44,54,1)"
+}
+
+interface Marker {
+    dot: WebdriverIO.Element;
+    label: WebdriverIO.Element;
+}
+
+export default class SliderWidget {
+    name: string;
+    element: WebdriverIO.Element;
+
+    constructor(name) {
+        this.name = name;
+        this.element = page.getWidget(this.name);
+    }
+
+    getRoot(): WebdriverIO.Element {
```

Comment:
maybe we can use arrow funcs ?

Reply:
What would be the benefit of that?

Reply:
I double checked and i think for this case it will require functions to declare as const. so the current one is good to go :)